packages/loader/container-loader/src/container.ts

```diff
@@ -291,6 +300,9 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private service: IDocumentService | undefined;
     private _connectionState = ConnectionState.Disconnected;
     private readonly _audience: Audience;
+    private deltaStorageDelay: number = 0;
```

Comment:
can we put these 3 into an array, and use index to access (i.e. RetryFor has values 0..2)?

Reply:
Removed it.